bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -473,9 +473,9 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- The default filenames are given relative to the root directory
      for the CLM2 data in the CESM distribution -->
 <!-- Plant function types (relative to {csmdata}) -->
-<paramfile phys="clm5_1" >lnd/clm2/paramdata/ctsm51_params.c210208.nc</paramfile>
-<paramfile phys="clm5_0" >lnd/clm2/paramdata/clm50_params.c210208.nc</paramfile>
-<paramfile phys="clm4_5" >lnd/clm2/paramdata/clm45_params.c210208.nc</paramfile>
+<paramfile phys="clm5_1" >lnd/clm2/paramdata/ctsm51_params.c210429.nc</paramfile>
+<paramfile phys="clm5_0" >lnd/clm2/paramdata/clm50_params.c210418.nc</paramfile>
+<paramfile phys="clm4_5" >lnd/clm2/paramdata/clm45_params.c210429.nc</paramfile>
```

Comment:
In dev033 I updated the params files starting from the ones dated 210208.

From what I can tell, there may be new params files in later tags, so I may need to recreate my new ones.

Reply:
Making these new params files available to all cases, allowed the cheyenne test-suite to pass except for
**Clm45Cn, Clm50Cn, and Clm45Fates tests**

Which raises a question for @wwieder and @ekluzek :
I am not putting effort into the CN code, but what about Clm45Fates? Is Clm45 on its way out or should I try to figure out why these tests are not bfb with baseline?

Reply:
I'll let @ekluzek weigh in here, not sure how we're handling CLM45Fates, but it's not a configuration that I see being very useful moving forward.

Reply:
> @slevisconsulting yes there have been updates to the paramsfiles so you'll need to tack your changes onto the end of the latest. I have a script based around ncks that does this. Do you have an easy way to add these parameters on?

My method feels relatively easy and includes the use of ncks. In this case I'm adding two new variables so:
- I ncdump the most recent params file to a text file
- I introduce the new vars to the text file manually
- I ncgen from text to .nc and this is my intermediate file
- Now I use ncks to get the new vars from the intermediate file to exact copies of the most recent params files and these become the final files. So this part repeats for clm45, clm50, and ctsm51. I will have to repeat this part on the most-most recent params files :-)
`ncks -A -v initial_Cstocks_bgc clm50_params.c<interm_date>.nc clm45_params.c<final_date>.nc`
`ncks -A -v initial_Cstocks_depth_bgc clm50_params.c<interm_date>.nc clm45_params.c<final_date>.nc`
- I also did a couple of renames:
`ncrename -v tau_cwd,tau_cwd_bgc clm45_params.c<final_date>.nc`
`ncrename -v k_frag,k_frag_cn clm45_params.c<final_date>.nc`

Reply:
> @slevisconsulting note below there is a problem in the Clm45 params file that could be the bfb issue. Also the only Clm45Fates test that they are going to keep is this one: "ERS_Ld5" grid="f19_g17" compset="I2000Clm45Fates" testmods="clm/FatesColdDef

Ok, thank you @ekluzek

Reply:
> So this part repeats for clm45, clm50, and ctsm51. I will have to repeat this part on the most-most recent params files :-)

@ekluzek this is probably obvious, but just to keep correct records:
Based on your comments that the values of the two new parameters should differ in the clm45 params file, my statement about "repeating" the ncks commands for clm45 needs the clarification that I am making a separate intermediate file for the clm45 case.